libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -131,8 +131,8 @@ class AP_BattMonitor
     bool consumed_wh(float&wh, const uint8_t instance = AP_BATT_PRIMARY_INSTANCE) const WARN_IF_UNUSED;
 
     /// capacity_remaining_pct - returns the % battery capacity remaining (0 ~ 100)
-    virtual uint8_t capacity_remaining_pct(uint8_t instance) const;
-    uint8_t capacity_remaining_pct() const { return capacity_remaining_pct(AP_BATT_PRIMARY_INSTANCE); }
+    virtual bool capacity_remaining_pct(int8_t &percentage, uint8_t instance) const WARN_IF_UNUSED; 
```

Comment:
I think everyother case of instance support we have puts the instance as the first argument, rather then the second which makes this look odd compared to the rest of the code base.

Reply:
The three functions above this take instance as the second argument, should I still make this one take it as first argument anyway?